example/.gitignore

```diff
@@ -54,3 +54,6 @@ buck-out/
 
 # Bundle artifact
 *.jsbundle
+
+msbuild.binlog
```

Comment:
you can just have *.binlog at the top instead since we would never want to check this file type in anyway